selfdrive/controls/lib/alerts_offroad.json

```diff
@@ -1,52 +1,40 @@
 {
   "Offroad_TemperatureTooHigh": {
-    "text": "Device temperature too high. System won't start.",
     "severity": 1
   },
   "Offroad_ConnectivityNeededPrompt": {
-    "text": "Immediately connect to the internet to check for updates. If you do not connect to the internet, openpilot won't engage in ",
     "severity": 0,
     "_comment": "Append the number of days at the end of the text"
   },
   "Offroad_ConnectivityNeeded": {
-    "text": "Connect to internet to check for updates. openpilot won't automatically start until it connects to internet to check for updates.",
     "severity": 1
   },
```

Comment:
I wonder, can we get along without this file now?

Reply:
I agree with you. this file should be mainly used in deprecated openpilot-apks to config & load offroad text. I think it's time to delete it.

Reply:
hmm, maybe put it in its own header file then? think that's cleaner than directly in the cc file